### DISCUSSION TITLE
假设一种情景

### DISCUSSION THREAD

**Discussion by neouks** [obsidianzh/forum#131] [Q&A常见问题]
我在写文章的时候通常会用到很多图片文件，而obsidian好像只可以把图片存到本地（我只想存到本地），但是存放的图片文件都是在一个目录里
<img width="993" alt="image" src="https://user-images.githubusercontent.com/26845888/173260879-608e7f46-5751-43d7-96ae-47f2880d4126.png">
假设我有写了很多文章里面引入了很多图片，而这些不同文章所用到的图片全部都在一个文件夹里，假设我需要保存一份md文件发给另外一个人，文章里的图片难道还需要我单独去找到然后拷贝出来吗，总不能因为一个文章把所有文章的图片都拷贝出来
需要一个像typora那样的功能，单独文章创建单独的资源文件夹，这样复制给别人也只需要复制md文件和资源文件夹
<img width="712" alt="image" src="https://user-images.githubusercontent.com/26845888/173261057-88e3bbea-26ce-4926-9082-969c918eb6a0.png">
<img width="298" alt="image" src="https://user-images.githubusercontent.com/26845888/173261142-43e55b42-574d-45fd-b776-9dd7521bb867.png">

PS：增加删除图片资源的选项
<img width="156" alt="image" src="https://user-images.githubusercontent.com/26845888/173261174-0b804b34-aa8b-4986-8ef0-69fd371e1363.png">

**Comment by lefinite** [comment]
和typora不一样，obsidian是作为知识库存在的，这样处理文件夹多了，目录结构也复杂了
要发给别人的话，要么弄个picgo把图片传到图床，要么就弄个发布服务。
发布服务除了官方的，也有第三方不用钱的，比如DigitalGarden：https://github.com/oleeskild/Obsidian-Digital-Garden
